### PR TITLE
Added delay after typing in Cypress test

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
addd delay after typing a character to avoid falky tests

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

File: app/client/cypress/support/commands.js [hunk 1]
```diff
@@ -859,7 +859,7 @@ Cypress.Commands.add("CreationOfUniqueAPIcheck", (apiname) => {
   cy.get(apiwidget.ApiName).click({ force: true });
   cy.get(apiwidget.apiTxt)
     .clear()
-    .type(apiname, { force: true })
+    .type(apiname, { delay: 500 })
```
Comment (hetunandu) [inline app/client/cypress/support/commands.js:862]: The unique name test failed in the last run. I don't think this fix is working

**Review by hetunandu — CHANGES_REQUESTED** [review]

File: app/client/cypress/support/commands.js (hunk 1 above)
Comment (NandanAnantharamu) [reply app/client/cypress/support/commands.js]: The latest code update is working can you check once

**Comment by arslanhaiderbuttar** [comment]
@NandanAnantharamu I ran the test cases and I am still having the stale exception on my local. Instead the putting the wait in the command, I have added the 3 seconds delay between CreationOfUniqueAPIcheck() in spec file "API_Unique_name_spec.js". This fixed the issue.

**Comment by NandanAnantharamu** [comment]
> @NandanAnantharamu I ran the test cases and I am still having the stale exception on my local. Instead the putting the wait in the command, I have added the 3 seconds delay between CreationOfUniqueAPIcheck() in spec file "API_Unique_name_spec.js". This fixed the issue.

The standard is to try not to add hard waits as much as possible, this test is working as is with current changes. Even in CI the test worked, we should check and then add hard waits.

**Review by arslanhaiderbuttar — APPROVED** [review]

**Review by arslanhaiderbuttar — APPROVED** [review]

**Review by hetunandu — APPROVED** [review]

**Review by arslanhaiderbuttar — APPROVED** [review]

**Review by arslanhaiderbuttar — DISMISSED** [review]

**Review by yatinappsmith — DISMISSED** [review]

**Comment by mohanarpit** [comment]
@NandanAnantharamu Is this PR still valid?

**Comment by NandanAnantharamu** [comment]
> @NandanAnantharamu Is this PR still valid?

This was to fix a flaky test, right now the test is working without a fix so we haven't pushed this PR. I will check once with the team and then take a call whether to close or push this one.

**Review by arslanhaiderbuttar — APPROVED** [review]

**Pull request merged**